extensions/amp-access/0.1/iframe-api/package.json

```diff
@@ -4,8 +4,21 @@
   "description": "IFrame API for amp-access",
   "author": "The AMP HTML Authors",
   "license": "Apache-2.0",
+  "main": "build/index.min.js",
```

Comment:
Let's also update https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/iframe-api/README.md

Reply:
what exactly do we need to update there?